src/yggdrasil/debug.go

```diff
@@ -257,8 +259,8 @@ func DEBUG_wire_encode_coords(coords []byte) []byte {
 
 func (c *Core) DEBUG_getDHTSize() int {
```

Comment:
If we are going to rewrite the sim, is this function needed? I suspect a lot of what's in `debug.go` isn't needed anymore.